src/components/AccessibilityWarning/AccessibilityWarningCard.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC, ReactElement } from 'react'
+
+import { AccessibilityWarning } from '@app-a11y/entities'
+import { Icon } from '@app-components'
+
+import './AccessibilityWarningCard.scss'
+
+export interface AccessibilityWarningCardProps {
+    warnings: (AccessibilityWarning | null)[]
```

Comment:
You probably should write this as:

```tsx
warnings: AccessibilityWarning[] | null,
```

...as it won't be an array of `null` values, right?

Reply:
Might no longer be relevant, see related other comment